pages/style.css

```diff
@@ -0,0 +1,75 @@
+@import url("/utils/color.css");
+@import url("/utils/reset.css");
```

Comment:
would suggest to keep this in the top in the import list, as it might override some wanted other files above

Reply:
done